### ISSUE TITLE
Event hub namespace now support public access enabled attribute

### ISSUE THREAD

**Issue by omerfeldman2** [hashicorp/terraform-provider-azurerm#17483]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Event hub namespace now support public access enabled attribute
Link to docs:
https://docs.microsoft.com/en-us/azure/templates/microsoft.eventhub/namespaces?tabs=json

### New or Affected Resource(s)/Data Source(s)

Azurerm_event_hub_namespace

### Potential Terraform Configuration

```hcl
resource "azurerm_eventhub_namespace" "example" {
  name                = "example-namespace"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  sku                 = "Standard"
  capacity            = 2

  Public_access_enabled = false
  tags = {
    environment = "Production"
  }
```

### References

https://docs.microsoft.com/en-us/azure/templates/microsoft.eventhub/namespaces?tabs=json

**Comment by xiaxyi** [comment]
@omerfeldman2 Thanks for this, I'll work on it.

**Comment by omerfeldman2** [comment]
> @omerfeldman2 Thanks for this, I'll work on it.

Do you know when it will be released? No pressure just wondering 😄

**Comment by xiaxyi** [comment]
@omerfeldman2 There is no ETA yet but it's in my backlog, I'll try update soon.

**Comment by xiaxyi** [comment]
@omerfeldman2 you can track the status in the mentioned pr

**Comment by katbyte** [comment]
closed by #17194
